src/agent/protocols/protos/image.proto

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (c) 2021 Alibaba Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+syntax = "proto3";
+
+option go_package = "github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/agent/protocols/grpc";
+
+package grpc;
+
+// Image defines the public APIs for managing images.
+service Image {
+    // PullImage pulls an image with authentication config.
+    rpc PullImage(PullImageRequest) returns (PullImageResponse) {}
+}
+
+message PullImageRequest {
+    // Name of the image.
+	string image = 1;
+	string container_id = 2;
```

Comment:
Can you please explain why container_id is needed for pulling image ?

Reply:
This field was moved from CCv0 to here, which I feel is not used.

Reply:
When running from k8s/crictl, we should not need a this one.